.github/workflows/ci.yml

```diff
@@ -19,95 +22,95 @@ jobs:
 
         include:
           # Linux
-          - toolchain: linux-gcc
+          - toolchain: linux-gcc-out-of-tree
```

Comment:
This and all similar changes seem unnecessary in this commit (especially since they are removed by the following https://github.com/skvadrik/re2c/pull/319/commits/9dbcee51b5f7ec49effd0c3a0fc0aca92a9e0704), can you remove them?

Reply:
I'll squash commits at the end. Is this fits?

Reply:
Sure, although I think it's better to keep unrelated changes as separate commits. But squashing is also fine, as long as the commit message lists the individual changes.